### PR TITLE
bpo-37483: Add PyObject_CallOneArg() in the What's New in Python 3.9

### PR DESCRIPTION
**Pull request by corona10**
https://bugs.python.org/issue37483

### CODE REVIEW

**Review by vstinner — CHANGES_REQUESTED** [review]
Private function should not be documented. Only public function should be documented.

File: Doc/whatsnew/3.9.rst [hunk 1]
```diff
@@ -1289,6 +1289,10 @@ New Features
   representation of a function-like object.
   (Patch by Jeroen Demeyer in :issue:`37645`.)
 
+* Added :c:func:`_PyObject_CallOneArg` for calling an object with one
```
Comment (vstinner) [inline Doc/whatsnew/3.9.rst]: Oh wait, you mean the public https://docs.python.org/3.9/c-api/call.html#c.PyObject_CallOneArg ?

You're right, this one is part of the public C API. I wrote a compatibility implementation for Python 3.8 and older in https://github.com/pythoncapi/pythoncapi_compat ;-)

**Comment by vstinner** [comment]
PR title:

>  bpo-37483: Add missing API in the What's New in Python 3.9 #23062 

I suggest:

bpo-37483: Add PyObject_CallOneArg() in the What's New in Python 3.9

File: Doc/whatsnew/3.9.rst (hunk 1 above)
Comment (corona10) [reply Doc/whatsnew/3.9.rst]: @vstinner Oops. sorry it was PyObject_CallOneArg  ;) I updated the PR

**Comment by corona10** [comment]
I have made the requested changes; please review again

**Review by vstinner — APPROVED** [review]
LGTM.

**Pull request merged**